package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "analetica-backend",
+  "version": "1.0.0",
+  "description": "backend of the webiste",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We can add another script here to start the app, `"start": "node src/index.js"`

On a slightly unrelated note, take a look at `nodemon`, it might come in handy to you.

Reply:
Sir, as the dependency nodemon will automatically get installed whenever someone will initialize the app. So, Should I change the start script to `"start": "nodemon src/index.js"`?